rebar.config

```diff
@@ -20,8 +20,9 @@
 {deps, []}.
 
 {project_plugins, [
-    {rebar3_hex, "6.11.4"},
-    {rebar3_lint, "0.4.0"}
+    rebar3_hank,
+    rebar3_hex,
+    rebar3_lint
```

Comment:
^ (I've been noticing the versionless import of plugins throughout this PR series and it does make me uneasy. *I know* it will bite us back one day.)

Reply:
They're not really important for the maintenance of this, and not having to care about that is really nice. Also, if a local `make version` complains, we can then lock.